tests/whitebox/limit/wb_enforcer.cc

```diff
@@ -30,6 +30,7 @@
 "  \"name\": \"CAS POST TEST COORDINATOR CONF-c3b05b0a-ae93-4aa6-b804-72d31137ac3f\","\
 "  \"type\": \"ENFORCER\","\
 "  \"customer_id\": \"DEADDEAD\","\
+"  \"account_type\": \"P\","\
```

Comment:
can remove all the changes in this file too.. account_type will not be part of enforcer..